client/app/account/billing/payment/method/billing-payment-method.html

```diff
@@ -118,10 +118,10 @@ <h2 data-translate="billing_payment_method_title"></h2>
             </oui-action-menu>
 
             <extra-top>
-                <a class="oui-button oui-button_secondary"
-                   data-ui-sref="app.account.billing.payment.method.add"
-                   data-translate="billing_payment_method_table_top_add_method">
-                </a>
+                <oui-button data-variant="secondary"
+                   data-on-click="$ctrl.goToAddPayment()"
+                   data-text="{{ :: 'billing_payment_method_table_top_add_method' | translate }}">
```

Comment:
Prefer adding a `<span data-translate="billing_payment_method_table_top_add_method"></span>` inside the `<oui-button>` element instead.

Beause the `text` attribute is deprecated.

ref: https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/oui-angular/button#deprecated

Thanks!